### PR TITLE
Delete attachments

### PR DESCRIPTION
**Pull request by CorneeSean**

### CODE REVIEW

File: frontend/src/app/shared/components/confirmation-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+      .mat-dialog-actions {
+        padding: 10px 0;
+      }
+    `,
+  ],
+})
+export class ConfirmationDialog {
+
+  constructor(@Inject(MAT_DIALOG_DATA) private data: ConfirmationDialogData = {}) {}
```
Comment (chodorowicz) [inline frontend/src/app/shared/components/confirmation-dialog.component.ts:51]: nit: spacing between lines is a bit strange in this class

File: frontend/src/app/shared/components/confirmation-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+@Component({
+  selector: 'eko-confirmation-dialog',
+  template: `
+    <h2 mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content class="mat-typography"
+      >{{ contentText }}
+      <ng-content></ng-content>
+    </mat-dialog-content>
+    <mat-dialog-actions align="end">
+      <button
```
Comment (chodorowicz) [inline frontend/src/app/shared/components/confirmation-dialog.component.ts:20]: For me having confirm/yes action on the right feels much more natural for the web https://ux.stackexchange.com/a/63239/42792 (OK on the left is more of Windows convention)

File: frontend/src/app/shared/components/confirmation-dialog.component.ts (hunk 1 above)
Comment (CorneeSean) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts]: How about now?

File: frontend/src/app/shared/components/confirmation-dialog.component.ts (hunk 2 above)
Comment (CorneeSean) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts:20]: Hmm, I really haven't been thinking about this topic much. That is defenitely an interesting thread. I am using a Windows a lot lately and it has subconscious influence as it seems. Interestingly, when I wrote the first version of confirmation dialog for delete, I was not using Windows at that time, I left my job as a contractor at Google just a few months back and I placed the confirmation button ON THE RIGHT side (see the deleted dialog code) . So it is basically Google vs Microsoft fight, right (just kidding)? Let's choose Google then :)

Comment (chodorowicz) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts:23]: Yeah, like scroll direction 😉  But I think right side confirm is more popular on the internet. Even Microsoft->Github is using right side on the web. Or is it switched on Windows?
image

File: frontend/src/app/shared/components/confirmation-dialog.component.ts (hunk 1 above)
Comment (chodorowicz) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts]: 👍

File: frontend/src/app/shared/components/confirmation-dialog.component.ts (hunk 2 above)
Comment (CorneeSean) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts:23]: image

Comment (CorneeSean) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts:23]: Weird, I don't have them separated but comment is still on the right

Comment (CorneeSean) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts:23]: @chodorowicz  Chrome alert is on the left, thoguh
image

Comment (chodorowicz) [reply frontend/src/app/shared/components/confirmation-dialog.component.ts:23]: yeah, the warning is a system browser/system notice

**Pull request merged**